pkg/api/cronworkflow/v1/types.go

```diff
@@ -102,7 +100,7 @@ type CronWorkflowStatus struct {
 type WorkflowTemplateSpec struct {
 	// Standard object's metadata of the workflows created from this template.
 	// More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata
-	// +optional
+	// +optionalfr
```

Comment:
is this change a typo? what does mean `+optionalfr` ?

Reply:
cruft :)